nextcord/guild.py

```diff
@@ -3674,4 +3674,9 @@ async def create_automod_rule(
         return rule
 
     def get_automod_rules(self) -> List[AutoModerationRule]:
+        """List all auto moderation rule configurated for this server.
+
+        Returns
+        --------
+        List[:class:`AutoModerationRule`]"""
```

Comment:
You need a newline in-between the contents of the docstring and the `"""` symbol